File: Dynamic programming/fibonacci-modified/fibonacci-modified.js
function fibonacciModified(t1, t2, n) {
  fibArray = [t1, t2]
  for (let i = 1; i < n; i++) {
      for (let j = 0; j <= n; j++) {
        tx = fibArray[j] + parseFloat(Math.pow(fibArray[j + i], 2))
        fibArray.push(tx)
      }
  }
  return fibArray[n - 1] ;
}
console.log(fibonacciModified(0, 1, 10));

File: Dynamic programming/fibonacci-modified/fibonacci-modified.py
def fibonacciModified(t1, t2, n):
    fibArray = [t1, t2]
    for i in range(1, n):
        for j in range(0, n):
            tx = fibArray[j] + (fibArray[j + i] ** 2)
            fibArray.append(tx)
    print(fibArray[n - 1])

fibonacciModified(0, 1, 10)
